webui/react/.eslintrc.js

```diff
@@ -35,18 +36,19 @@ module.exports = {
       files: ['**/src/e2e/**'],
       rules: {
         '@typescript-eslint/no-floating-promises': 'error',
+        'no-console': 'off',
       },
     },
   ],
   parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaFeatures: { jsx: true },
     ecmaVersion: 2018,
+    project: true,
```

Comment:
precommit was failing without this

Reply:
reference purpose:
[reference of `parserOptions`](https://typescript-eslint.io/getting-started/typed-linting/monorepos/#one-tsconfigjson-per-package-and-an-optional-one-in-the-root)

Reply:
yeah i dont manage this stuff. replacing that line didn't work, and i just want precommit to stop shouting at me for changes someone else made. you can replace `project: true` in your own pr